### DISCUSSION TITLE
What is a first order function?

### DISCUSSION THREAD

**Discussion by montasim** [STRINGLABITSOLUTIONS/Bhojon#350] [Q&A] [answered]
In short.

**Comment by d4tdev** [comment, marked as answer]
First-order function is a function that accepts only arguments (arguments) of object data type or primitive data type and returns only values ​​of primitive data type or object. statue.

**Comment by Preetiraj3697** [comment]
Passing an anonymous function into an iterator like forEach already demonstrates some nice support for first-class objects—the ability to treat functions as variables that can be assigned and passed around.
